test/NCI.OCPL.Api.CTSListingPages.Tests/Tests/Controllers/ListingInformationControllerTest.GetByIds.cs

```diff
@@ -0,0 +1,213 @@
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Logging.Testing;
+using Moq;
+using Xunit;
+
+using NCI.OCPL.Api.Common;
+using NCI.OCPL.Api.CTSListingPages.Controllers;
+using System;
+
+namespace NCI.OCPL.Api.CTSListingPages.Tests
+{
+    public partial class ListingInformationControllerTest
+    {
+        /// <summary>
+        /// Verify correcting handling of invalid c-codes.
+        /// </summary>
+        [Theory]
+        [InlineData(new object[] { null })]
+        [InlineData(new object[] { new string[] { } })]       // Can't use String.Empty because it's a property instead of a constant.
+        public async void GetByIds_InvalidCCodes(string[] ccode)
+        {
```

Comment:
Updated! Included a variety of tests.

Reply:
You still have a comment mis-explaining that the empty array is there because String.Empty isn't a constant. The empty array is actually there because you're testing that an empty array is handled correctly.

Also consider whether it's worth adding a test for an array consisting solely of the empty string. (I good either way.)

Reply:
Added a few new tests and lots of comments describing the reasons for each!